react/features/deep-linking/functions.js

```diff
@@ -42,12 +47,22 @@ export function generateDeepLinkingURL() {
     // like to open the current URL in the mobile app. The only way to do it
     // appears to be a link with an app-specific scheme, not a Universal
     // Link.
+
     const appScheme = interfaceConfig.APP_SCHEME || 'org.jitsi.meet';
+    const href = window.location.href;
```

Comment:
How about deconstruction here?

Reply:
I don't find deconstruction particularly useful for single variables.